src/components/SettingsModal.js

```diff
@@ -1,17 +1,79 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import PropTypes from "prop-types";
 
 export default function SettingsModal({ setDisplayModal }) {
+  const DEFAULT_VALUE = 15;
+  const [sliderValue, setSliderValue] = useState(
+    localStorage.getItem("sliderValueInLocalStorage") || DEFAULT_VALUE
+  );
+
+  useEffect(() => {
+    localStorage.setItem("sliderValueInLocalStorage", sliderValue);
+  }, [sliderValue]);
+
+  const handleSliderChange = (e) => {
+    const { value } = e.target;
+    setSliderValue(value);
+  };
+
   return (
     <div id="modal" className="modal-container">
       <section className="modal-content">
-        <img
-          src={"/img/close-x.png"}
-          alt="close icon"
-          onClick={() => setDisplayModal(false)}
+        <button
           className="modal-close-button"
-        />
+          onClick={() => setDisplayModal(false)}
+        >
+          <img
+            src={"/img/close-x.png"}
+            alt="close icon"
+            className="modal-close-img"
+          />
+        </button>
```

Comment:
Was this added due to accessibility? If so, cool!

Reply:
Yes, exactly. When I was tabbing through the modal to test out VoiceOver, I had noticed it wasn't highlighting it.